src/Compilers/CSharp/Test/Emit/PDB/PDBLambdaTests.cs

```diff
@@ -89,115 +89,22 @@ public static int Main()
-        <entry offset=""0xe"" startLine=""14"" startColumn=""9"" endLine=""22"" endColumn=""11"" document=""1"" />
-        <entry offset=""0x1b"" startLine=""23"" startColumn=""9"" endLine=""23"" endColumn=""22"" document=""1"" />
-        <entry offset=""0x25"" startLine=""24"" startColumn=""5"" endLine=""24"" endColumn=""6"" document=""1"" />
-      </sequencePoints>
-      <scope startOffset=""0x0"" endOffset=""0x27"">
-        <local name=""CS$&lt;&gt;8__locals0"" il_index=""0"" il_start=""0x0"" il_end=""0x27"" attributes=""0"" />
-        <local name=""f1"" il_index=""1"" il_start=""0x0"" il_end=""0x27"" attributes=""0"" />
-      </scope>
-    </method>
-    <method containingType=""Test+&lt;&gt;c__DisplayClass1_0"" name=""&lt;M&gt;b__0"" parameterNames=""x"">
-      <customDebugInfo>
-        <forward declaringType=""Test"" methodName=""Main"" />
-        <encLocalSlotMap>
-          <slot kind=""30"" offset=""56"" />
-          <slot kind=""0"" offset=""110"" />
-          <slot kind=""21"" offset=""56"" />
-        </encLocalSlotMap>
-      </customDebugInfo>
-      <sequencePoints>
-        <entry offset=""0x0"" hidden=""true"" document=""1"" />
-        <entry offset=""0x14"" startLine=""15"" startColumn=""9"" endLine=""15"" endColumn=""10"" document=""1"" />
-        <entry offset=""0x15"" startLine=""16"" startColumn=""13"" endLine=""16"" endColumn=""23"" document=""1"" />
-        <entry offset=""0x1c"" startLine=""17"" startColumn=""13"" endLine=""20"" endColumn=""15"" document=""1"" />
-        <entry offset=""0x29"" startLine=""21"" startColumn=""13"" endLine=""21"" endColumn=""26"" document=""1"" />
-        <entry offset=""0x33"" startLine=""22"" startColumn=""9"" endLine=""22"" endColumn=""10"" document=""1"" />
-      </sequencePoints>
-      <scope startOffset=""0x0"" endOffset=""0x35"">
-        <local name=""CS$&lt;&gt;8__locals0"" il_index=""0"" il_start=""0x0"" il_end=""0x35"" attributes=""0"" />
-        <local name=""f2"" il_index=""1"" il_start=""0x0"" il_end=""0x35"" attributes=""0"" />
-      </scope>
-    </method>
-    <method containingType=""Test+&lt;&gt;c__DisplayClass1_1"" name=""&lt;M&gt;b__1"" parameterNames=""y"">
-      <customDebugInfo>
-        <forward declaringType=""Test"" methodName=""Main"" />
-        <encLocalSlotMap>
-          <slot kind=""21"" offset=""136"" />
-        </encLocalSlotMap>
-      </customDebugInfo>
-      <sequencePoints>
-        <entry offset=""0x0"" startLine=""18"" startColumn=""13"" endLine=""18"" endColumn=""14"" document=""1"" />
-        <entry offset=""0x1"" startLine=""19"" startColumn=""17"" endLine=""19"" endColumn=""38"" document=""1"" />
-        <entry offset=""0x1f"" startLine=""20"" startColumn=""13"" endLine=""20"" endColumn=""14"" document=""1"" />
-      </sequencePoints>
-    </method>
-  </methods>
-</symbols>");
+
+            TestLambdasAndClosures(source);
```

Comment:
>TestLambdasAndClosures [](start = 12, length = 22)

I am curious what would it be like to create a new test? How easy would it be to figure out what markup to add and where in order  to define the baseline? How easy would it be analyze baseline failures?
What is the real motivation behind the change? Did we consider an alternative of adding a helper that trims down the XML to relevant parts to decrease what is in the baseline instead?

Reply:
Stripping down XML makes it better but still not ideal since it's hard to correlate the numbers in the XML (syntax offsets) with the source. The goal is to make it easy to see that the test is correct.
As I commented [below](https://github.com/dotnet/roslyn/pull/50371#discussion_r555290540) the test should print the source with the markers if it fails. Then it'd be easy to author the test. You'd just run it once without markers (or rather wiht just the `method` start marker) and it would print the expected source that you can then copy & paste as expected marked source.